rholang/src/main/scala/coop/rchain/rholang/interpreter/compiler/normalizer/processes/PBundleNormalizer.scala

```diff
@@ -28,13 +28,20 @@ object PBundleNormalizer {
     def error(targetResult: ProcVisitOutputs): F[ProcVisitOutputs] = {
       val errMsg = {
         def at(variable: String, sourcePosition: SourcePosition): String =
-          variable + " line: " + sourcePosition.row + ", column: " + sourcePosition.column
+          variable + " at " + sourcePosition.row + ":" + sourcePosition.column
         val wildcardsPositions = targetResult.knownFree.wildcards.map(at("", _))
         val freeVarsPositions = targetResult.knownFree.levelBindings.map {
           case (n, LevelContext(_, _, sourcePosition)) => at(s"`$n`", sourcePosition)
         }
-        wildcardsPositions.mkString(" Wildcards at positions: ", ", ", ".") ++
-          freeVarsPositions.mkString(" Free variables at positions: ", ", ", ".")
+        val errMsgWildcards =
+          if (wildcardsPositions.nonEmpty)
+            wildcardsPositions.mkString(" Wildcards positions ", ", ", ".")
+          else ""
```

Comment:
Its shorter to write this as 
`wildcardsPositions.map(_.mkString(" Wildcards positions ", ", ", ".")).getOrElse("")`
you map into Option, transform value inside, and then getOrElse on this Option